README.md

```diff
@@ -34,11 +34,12 @@ High level diagram of the stack:
 The following packages are included in the Fury Kubernetes Logging katalog:
 
 | Package                                                | Version                         | Description                                                                                 |
-| ------------------------------------------------------ |---------------------------------| ------------------------------------------------------------------------------------------- |
+| ------------------------------------------------------ | ------------------------------- | ------------------------------------------------------------------------------------------- |
```

Comment:
I think cerebro should be removed here aswell

Reply:
nice catch! removed